react-native-fbsdklogin/js/FBSDKLoginButton.ios.js

```diff
@@ -29,10 +28,32 @@ var {
   StyleSheet,
 } = React;
 
+import type {
+  FBSDKLoginCallback,
+  FBSDKLoginBehavior,
+  FBSDKDefaultAudience,
+} from './FBSDKLoginManager.ios.js';
+
+type FBSDKLoginButtonProps = {
+  readPermissions: Array<string>;
+  publishPermissions: Array<string>;
+  onLoginFinished: FBSDKLoginCallback;
+  onLogoutFinished: () => void;
+  loginBehavior: FBSDKLoginBehavior,
+  defaultAudience: FBSDKDefaultAudience,
+  tooltipBehavior: $Enum<{
```

Comment:
what's the difference between using `$Enum` and `'auto' | 'force-display' | 'disable'`?